tasks/modules.py

```diff
@@ -356,3 +358,23 @@ def for_each(ctx: Context, cmd: str, skip_untagged: bool = False):
             continue
         with ctx.cd(mod.full_path()):
             ctx.run(cmd)
+
+
+@task
+def validate(_: Context, fail_fast: bool = False):
+    """
+    Test if every module was properly added in the DEFAULT_MODULES list.
+    """
+    missing_modules = []
+    for module in DEFAULT_MODULES.values():
+        for dependency in module.dependencies:
+            if dependency not in DEFAULT_MODULES:
+                if fail_fast:
+                    print(f"Error: {module.path} depends on missing {dependency}", file=sys.stderr)
+                    sys.exit(1)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏ Why not using `raise Exit` or `exit` ?

Reply:
because of copilot